openeo-geotrellis/src/main/scala/org/openeo/geotrellis/package.scala

```diff
@@ -56,7 +58,7 @@ package object geotrellis {
         .maxBackoffTime(Duration.ofMillis(15))
         .build()
     val retryPolicy =
-      RetryPolicy.defaultRetryPolicy()
+      software.amazon.awssdk.core.retry.RetryPolicy.defaultRetryPolicy()
```

Comment:
At the top:

```scala
import software.amazon.awssdk.core.retry.{RetryPolicy => AwsRetryPolicy}
import net.jodah.failsafe.{RetryPolicy => FailsafeRetryPolicy}
```

then:

```suggestion
      AwsRetryPolicy.defaultRetryPolicy()
```

Reply:
changed